terraform/modules/PersistGrantsGovXMLDB/main.tf

```diff
@@ -37,15 +37,17 @@ module "lambda_execution_policy" {
       ]
       resources = [
         data.aws_s3_bucket.prepared_data.arn,
-        # Path: <first 3 of grant id>/<grant id>/grants.gov/v2.xml
-        "${data.aws_s3_bucket.prepared_data.arn}/*/*/grants.gov/v2.xml"
+        # Path: <first 3 of grant id>/<grant id>/grants.gov/v2.xml (deprecated)
```

Comment:
if this is deprecated do we still need to have permissions to this bucket? Same question for `SplitGrantsGovXMLDB` as well.

Reply:
This is still needed for `PersistGrantsGovXMLDB` because the bucket has existing objects (which we aren't planning to move/recreate). 

However, `SplitGrantsGovXMLDB` no longer has any reason to have `GetObject` or `ListBucket` permissions for the bucket as it's write-only (only uses `PutObject`) going forward. I'll remove the permissions for that function.

Reply:
Just to add: Even though during normal pipeline execution it should only encounter objects with the new key convention, I'd like to keep the ability for `PersistGrantsGovXMLDB` to read existing S3 objects in case there's a bug where we need to trigger the Lambda to reprocess an object from S3 as a recovery step, or if we need to rebuild the DynamoDB index. After this is live and confirmed stable in Production for a few weeks, we should circle back to remove the permissions here as well. I'll make a follow-up issue for that.